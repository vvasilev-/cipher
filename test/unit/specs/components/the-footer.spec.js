/**
 * The external dependencies.
 */
import { shallow } from 'vue-test-utils';
import { createRenderer } from 'vue-server-renderer';

/**
 * The internal dependencies.
 */
import TheFooter from 'components/the-footer';

describe('TheFooter', () => {
	it('should render properly', async () => {
		const wrapper = shallow(TheFooter);
		const renderer = createRenderer();
		const result = await renderer.renderToString(wrapper.vm);

		expect(result).toMatchSnapshot();
	});
});
