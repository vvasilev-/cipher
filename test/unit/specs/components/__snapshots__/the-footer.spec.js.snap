// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter should render properly 1`] = `"<footer data-server-rendered=\\"true\\" class=\\"mt4 tc\\"><a href=\\"#\\" class=\\"dib light-silver no-underline mh2 dim\\"><!----></a></footer>"`;
